pkgs/tools/compression/lzop/default.nix

```diff
@@ -12,6 +14,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     homepage = "http://www.lzop.org";
     description = "Fast file compressor";
+    maintainers = with maintainers; [ ];
```

Comment:
Why add all these empty lists?

Reply:
so that people will visually see that someone should care for the package. If it is absent it can be easily overlooked.